tests/realtikvtest/pessimistictest/pessimistic_test.go

```diff
@@ -2913,6 +2913,40 @@ func TestChangeLockToPut(t *testing.T) {
 	tk.MustExec("admin check table t1")
 }
 
+func TestIssue28011(t *testing.T) {
```

Comment:
We don't have a test to cover the effect of changing, maybe it's needed to tested by hand.